RecoLocalTracker/SiPixelRecHits/plugins/alpaka/PixelRecHits.h

```diff
@@ -21,7 +21,7 @@
 #include "HeterogeneousCore/AlpakaInterface/interface/workdivision.h"
 #include "RecoLocalTracker/SiPixelRecHits/interface/pixelCPEforDevice.h"
 
-//#define GPU_DEBUG
+#define GPU_DEBUG
```

Comment:
please comment this out

Reply:
I was working on rebasing and will resolve them

Reply:
OK, thanks.

In general, I would suggest to do the rebase locally and push it once you are done, to avoid comments from the bot and the reviewers on the intermediate steps.